dashboard/app/controllers/api/v1/pd/applications_controller.rb

```diff
@@ -1,4 +1,6 @@
 class Api::V1::Pd::ApplicationsController < ::ApplicationController
+  include Api::CsvDownload
```

Comment:
Note - the quick_view action isn't available yet so I will add the actual CSV download code once that's available in a followup.